src/components/sidebar/components/sidebar-filter-input/sidebar-filter-input.module.css

```diff
@@ -48,9 +50,6 @@
   composes: g-focus-ring-from-box-shadow from global;
   background-color: transparent;
   border: none;
-  border-radius: 5px;
-  box-shadow: inset 0 0 0 1px rgba(101, 106, 118, 0.3),
-    inset 0 1px 2px 1px rgba(101, 106, 118, 0.1);
   color: var(--token-color-palette-neutral-700);
   font-size: 14px;
```

Comment:
Wanna change the font size here to 16px? This will prevent iOS from zooming in a bit when focusing the input.